backend/mora/service/engagement.py

```diff
@@ -63,10 +61,21 @@ def create_engagement(req):
         opgaver=[{'uuid': job_function_uuid}] if job_function_uuid else []
     )
 
-    return c.organisationfunktion.create(engagement)
+    return {
```

Comment:
In order to generalise this to addresses, you should probably return the path and/or scope as well.